code/hud/hudconfig.cpp

```diff
@@ -1708,10 +1708,9 @@ void hud_config_color_load(const char *name)
 {
 	int idx;
 	char str[1024];
-	char *fname;
 	ubyte r, g, b, a;
 
-	fname = cf_add_ext(name, ".hcf");
+	auto fname = cf_add_ext(name, ".hcf");
```

Comment:
`const auto`? Not required but makes it easier to tell it's `const`.

Reply:
IMHO `const char *` is easier to read and not much more text than `auto` but seems like the latter is SCP standard.

Reply:
At one point our CI would complain about long type names where `auto` would suffice, but that's currently disabled.  I'm fine with putting `const char *` here.